build.gradle

```diff
@@ -45,8 +48,20 @@ task karma(type: NodeTask) {
   args = ["start", "--single-run", "--reporters", "progress,jenkins"]
 }
 
+task generateVersionFile << {
+  def buildInfo = [
+    version: project.hasProperty('deckVersion') ? "${deckVersion}" : "n/a",
+    created: new Date().getTime()
+  ]
+  def buildJson = JsonOutput.prettyPrint(JsonOutput.toJson(buildInfo))
+  new File(Paths.get("build", "webpack").toString()).mkdirs()
+  new File(Paths.get("build", "webpack", "version.json").toString()).write(buildJson)
+  new File("version.json").write(buildJson)
+}
```

Comment:
@cfieber will this even do what I think it will do in the end?

The goal is to get the `version.json` file available to the webpack build (by putting it at the root), but also getting it into apache at the root (by putting it in `build/webpack`, which the `ospackage` closure references)